src/main/java/bio/terra/cli/command/Main.java

```diff
@@ -159,6 +160,11 @@ public int handleExecutionException(Exception ex, CommandLine cmd, ParseResult p
             systemAndUnexpectedErrorStyle.errorText("[ERROR] ").concat(errorMessage);
         exitCode = SYSTEM_EXIT_CODE;
         printPointerToLogFile = true;
+      } else if (ex instanceof PassthroughException) {
+        errorMessage = ex.getMessage() == null ? "" : ex.getMessage();
```

Comment:
Out of interest, why not `errorText` color scheme here, like `UserActionable` and `System`?

Reply:
Originally I was thinking to distinguish between app errors and CLI errors, but on second thought I don't think that really makes sense. An error is an error to the CLI user, they don't care where it came from. Changed to use `errorText`.